examples/external-otlp-tonic-tokio/src/main.rs

```diff
@@ -47,7 +47,7 @@ fn init_tracer() -> Result<sdktrace::Tracer, TraceError> {
         .map(|(name, value)| {
             let header_name = name
                 .strip_prefix(HEADER_PREFIX)
-                .map(|h| h.replace("_", "-"))
+                .map(|h| h.replace('_', "-"))
```

Comment:
It's a little ugly to have this in a commit whose message is "change log".

Reply:
Sorry I didn't clean up the commit logs. Will force push a cleaned up version